build.gradle

```diff
@@ -92,3 +94,20 @@ task integrationTest(type: Test) {
 }
 
 check.dependsOn integrationTest
+
+spotbugsTest.enabled = false
+spotbugsIntTest.enabled = false
+spotbugsMain {
+	reports {
+		html {
+			enabled = false
+		}
+		xml {
+			enabled = true
+		}
+	}
+}
+
+checkstyle {
+	configFile = file("${rootDir}/config/checkstyle/checkstyle.xml")
```

Comment:
```suggestion
	configFile = file('${rootDir}/config/checkstyle/checkstyle.xml')
```

Reply:
In this case, the double quotes are needed because single quotes treat the `${rootDir}` variable as a part of the string as opposed to a value that needs to be interpolated into the string